src/components/Dashboard/TodoList.vue

```diff
@@ -603,15 +648,47 @@
             <PaymentPending v-else />
           </template>
 
+          <!-- is this an NR approved continuation draft item? -->
```

Comment:
Let's call this a "continuation authorization item".

Reply:
Line 651 - 661 are not needed, I will remove them. This is already handled in line 1613

Reply:
OK, but if you see an opportunity to update the comments, please call this filing in Draft, Pending, Change Requested statuses a "continuation authorization", while the later statuses are a "continuation application". This will be consistent with what they're called in Create UI and elsewhere.